clippy_lints/src/utils/internal_lints/metadata_collector.rs

```diff
@@ -509,7 +520,12 @@ fn extract_attr_docs(cx: &LateContext<'_>, item: &Item<'_>) -> Option<String> {
                 continue;
             }
         }
-        docs.push_str(line);
+        // This removes the leading space that the macro translation introduces
+        if let Some(stripped_doc) = line.strip_prefix(' ') {
+            docs.push_str(stripped_doc);
+        } else if !line.is_empty() {
+            docs.push_str(&line);
+        }
```

Comment:
The rebase might make it difficult to see where and how the conflicts were resolved. I added this block to strip the leading space on every filled line and added a doc comment to the method itself, the rest stayed the same. I also removed the JS implementation of the language extraction in a new commit.